soh/src/code/z_parameter.c

```diff
@@ -3129,7 +3133,7 @@ void Interface_DrawMagicBar(GlobalContext* globalCtx) {
                 gDPSetPrimColor(OVERLAY_DISP++, 0, 0, R_MAGIC_FILL_COLOR(0), R_MAGIC_FILL_COLOR(1), R_MAGIC_FILL_COLOR(2), interfaceCtx->magicAlpha);
             }
 
-            gDPLoadMultiBlock_4b(OVERLAY_DISP++, gMagicBarFillTex, 0, G_TX_RENDERTILE, G_IM_FMT_I, 16, 16, 0,
+            gDPLoadMultiBlock_4b(OVERLAY_DISP++, gMagicBarFillTex, 0, G_TX_RENDERTILE, G_IM_FMT_I, 8, 8, 0,
```

Comment:
This looks like an XML problem.  This is looking for an I4 but ZAPD thinks its an IA8.

Reply:
I mean the XMLs probably have the wrong texture type. I would need to check in decomp to make sure using I4 in the XML still matches.

Reply:
This should also not be changed from the authentic values. We may not export the whole texture due to bad asset XMLs.